### ISSUE TITLE
Unclear rejection messages in master

### ISSUE THREAD

**Issue by laanwj** [bitcoin/bitcoin#6862]
Since the LowS enforcement, we get messages like this with `-debug=mempoolrej` and 'sendrawtransaction':

    2015-10-21 08:44:56 65fcd08dbfa3e806f62b7eb1e9a9e21070e88e72f0b41ed4d83afa4e71b63d50 from peer=58 was not accepted: non-mandatory-script-verify-flag (No error) (code 64)

The message could at least mention what non-mandatory script-verify flag. We know, but other users may not.

**Comment by paveljanik** [comment]
Similar case today:
```
ERROR: AcceptToMemoryPoolWorker: CheckInputs: f62430326609f0ad79f9a7503ed59ede0ca6991513829337e4a210d22d1e618f, non-mandatory-script-verify-flag (unknown error) (code 64)
```

bc.info said High S... No other blockexplorer knows about the tx, so can't check myself.

**Comment by sipa** [comment]
Was this not fixed in 7276?

**Comment by sipa** [comment]
Ah, @paveljanik's case is another bug.

**Comment by laanwj** [comment]
Yes the issue seems to be different, "No error" versus "unknown error"

**Comment by sipa** [comment]
I think the original issue here is fixed by #7276. I'm working on a fix for @paveljanik's bug.
